style/gallery.less

```diff
@@ -57,6 +57,36 @@
 		}
 	}
 
+	&-slider {
+		display: inline-flex;
+		overflow: scroll;
+		scroll-snap-type: x mandatory;
+		scroll-behavior: smooth;
```

Comment:
Safari doesn't support this behaviour 😦 
https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior#Browser_compatibility